core/src/main/java/org/sonarsource/sonarlint/core/container/standalone/rule/StandaloneRuleParamType.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * SonarLint Core - Implementation
+ * Copyright (C) 2009-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarsource.sonarlint.core.container.standalone.rule;
+
+import org.sonar.api.server.rule.RuleParamType;
+
+public enum StandaloneRuleParamType {
+
+  /**
+   * Keep in sync with constants in {@link RuleParamType}
+   */
+  STRING, TEXT, BOOLEAN, INTEGER, FLOAT, SINGLE_SELECT_LIST, MULTI_SELECT_LIST;
+
+  static StandaloneRuleParamType from(RuleParamType apiType) {
+    if (apiType.multiple()) {
```

Comment:
I disagree with this mapping. Even for a multi-select list, we still need to know if elements are string, int, booleans...

Reply:
IMO `multiple` should be exposed the same way as `possibleValues`, in a different field.

Reply:
I agree. I misunderstood the range of possibilities offered by the plugin API.